src/main/java/io/jsonwebtoken/JwtParser.java

```diff
@@ -125,6 +125,15 @@
     JwtParser require(String claimName, Object value);
 
     /**
+     * Replaces the system time-of-day call to return this fixed value instead. A {@code null} value will remove
+     * the fixed date output and return the system time-of-day instead.
+     *
+     * @param now the time-of-day to use for parsimg this JWT or {@code null} to use the current time-of-day
+     * @return the builder instance for method chaining.
+     */
+    JwtParser setFixedClock(Date now);
```

Comment:
And we'd need a `DefaultClock` implementation would just return `new Date();` in its `now()` implementation.